.mocharc.yml

```
@@ -1 +1,7 @@
-spec: "test/**/*.spec.js"
+extension: ["ts"]
+spec: "test/**/*.spec.ts"
+require:
+ - "chai/register-should.js"
+ - "./test/setup.ts"
+node-option:
+  - "import=tsx"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 1 but found 2

(indentation)